tests/components/unifi/test_controller.py

```diff
@@ -68,10 +81,10 @@ async def setup_unifi_integration(
     controllers=None,
 ):
     """Create the UniFi controller."""
-    assert await async_setup_component(hass, unifi.DOMAIN, {})
+    assert await async_setup_component(hass, UNIFI_DOMAIN, {})
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5403678)